src/main/resources/application.conf

```diff
@@ -550,7 +550,36 @@ mantis {
 
       # Force checksum verification of all data that is read from the file system on behalf of a particular read
       verify-checksums = true
+
+      # This ensures that only 32 open files can be accessed at once
+      max-open-files = 32
+    }
+
+    rocksdb {
+      # RocksDB data directory
+      path = ${mantis.datadir}"/rocksdb/"
+
+      # Create DB data directory if it's missing
+      create-if-missing = true
+
+      # Should the DB raise an error as soon as it detects an internal corruption
+      paranoid-checks = true
+
+      # This ensures that only one thread will be occupied
+      max-threads = 1
+
+      # This ensures that only 32 open files can be accessed at once
+      max-open-files = 32
+
+      # Force checksum verification of all data that is read from the file system on behalf of a particular read
+      verify-checksums = true
+
+      # Can be turned on to make the write operation not return until the data being written has been pushed all the way to persistent storage
+      synchronous-writes = false
```

Comment:
Why exposing that option ? i mean it is probably performance killer, and we would not enable it ever.